liubai-docs/docs/guide/markdown-support.md

```
@@ -0,0 +1,72 @@
+
+![inline code](./assets-md-support/01.png)
+
+### 粗体
+
+在编辑器上，输入 ** + 你的粗体文字 + ** 即可包裹粗体。
+
+![bold](./assets-md-support/02.png)
+
+### 斜体
+
+在编辑器上，输入 * + 你的斜体文字 + * 即可包裹斜体。
```

markdownlint-cli2 (0.17.2)

Spaces inside emphasis markers
null

(MD037, no-space-in-emphasis)